plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -127,7 +127,20 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 	}
 
 	trace.Step("Selecting host")
-	return g.selectHost(priorityList)
+	host, err := g.selectHost(priorityList)
+	if err != nil {
+		return "", err
+	}
+
+	if len(g.extenders) != 0 {
+		for _, extender := range g.extenders {
+			err := extender.Bind(pod, host)
```

Comment:
That is a bigger change IMO and doesn't work with multiple extenders. So far, the extender implementation has left the primary interaction between scheduler and apiserver untouched.

Is your goal to start kubelet on the node where the pod is scheduled? It can be achieved by watching the apiserver for pod.Spec.NodeName != ""?